Mathlib/AlgebraicTopology/SimplicialSet/HomotopyCat.lean

```diff
@@ -349,5 +351,50 @@ def hoFunctor : SSet.{u} ⥤ Cat.{u, u} := SSet.truncation 2 ⋙ Truncated.hoFun
 end
 
 end
+section
+
+/-- Since `⦋0⦌ : SimplexCategory` is terminal, `Δ[0]` has a unique point and thus
+`OneTruncation₂ ((truncation 2).obj Δ[0])` has a unique inhabitant. -/
+instance instUniqueOneTruncation₂DeltaZero : Unique (OneTruncation₂ ((truncation 2).obj Δ[0])) :=
+  inferInstanceAs (Unique (ULift.{_, 0} (⦋0⦌ ⟶ ⦋0⦌)))
+
+/-- Since `⦋0⦌ : SimplexCategory` is terminal, `Δ[0]` has a unique edge and thus the homs of
+`OneTruncation₂ ((truncation 2).obj Δ[0])` have unique inhabitants. -/
+instance (x y : OneTruncation₂ ((truncation 2).obj Δ[0])) : Unique (x ⟶ y) where
+  default := by
+    obtain rfl : x = default := Unique.uniq _ _
+    obtain rfl : y = default := Unique.uniq _ _
+    exact 𝟙rq instUniqueOneTruncation₂DeltaZero.default
+  uniq _ := by
+    letI : Subsingleton (((truncation 2).obj Δ[0]).obj (.op ⦋1⦌₂)) :=
+      inferInstanceAs (Subsingleton (ULift.{_, 0} (⦋1⦌ ⟶ ⦋0⦌)))
+    ext
+    exact this.allEq _ _
+
+/-- The category `hoFunctor.obj (Δ[0])` is terminal. -/
+def hoFunctorDeltaZeroIsTerminal : IsTerminal (hoFunctor.obj (Δ[0])) := by
```

Comment:
This would be more consistent with our naming conventions:
```suggestion
def isTerminalHoFunctorDeltaZero : IsTerminal (hoFunctor.obj (Δ[0])) := by
```